pants.toml

```diff
@@ -6,6 +6,7 @@ pythonpath = ["%(buildroot)s/pants-plugins"]
 backend_packages.add = [
   "pants.backend.python",
   "pants.backend.experimental.python.lint.autoflake",
+  "pants.backend.python.docs.sphinx",
```

Comment:
I would expect new backends like this one to start off as experimental..?

Reply:
True. I hope to have it be stable soon, but can always graduate later.